### ISSUE TITLE
LLaDA-MoE-7B-A1B-Instruct.i1-Q4_K_M-GGUF send_error: task id = 0, error: the current context does not logits computation. skipping

### ISSUE THREAD

**Issue by grigio** [ggml-org/llama.cpp#17249]
Hi try tried to load this model but i get this error
...
llama-swap  |         {{- content }}
llama-swap  |         {{- '\n</tool_response>' }}
llama-swap  |         {%- if loop.last or (messages[loop.index0 + 1].role != "tool") %}
llama-swap  |             {{- '<|role_end|>' }}
llama-swap  |         {%- endif %}
llama-swap  |     {%- endif %}
llama-swap  | {%- endfor %}
llama-swap  | {%- if add_generation_prompt %}
llama-swap  |     {{- '<role>ASSISTANT</role>' }}
llama-swap  | {%- endif %}, example_format: '<role>SYSTEM</role>You are a helpful assistant
llama-swap  | detailed thinking off<|role_end|><role>HUMAN</role>Hello<|role_end|><role>ASSISTANT</role>Hi there<|role_end|><role>HUMAN</role>How are you?<|role_end|><role>ASSISTANT</role>'
llama-swap  | main: server is listening on http://0.0.0.0:9999 - starting the main loop
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  log_server_r: request: GET /health 127.0.0.1 200
llama-swap  | [INFO] <LLaDA-MoE-7B-A1B-Instruct.i1-Q4_K_M> Health check passed on http://127.0.0.1:9999/health
llama-swap  | srv  params_from_: Chat format: Hermes 2 Pro
llama-swap  | slot get_availabl: id  3 | task -1 | selected slot by LRU, t_last = -1
llama-swap  | slot launch_slot_: id  3 | task -1 | sampler chain: logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist 
llama-swap  | slot launch_slot_: id  3 | task 0 | processing task
llama-swap  | slot update_slots: id  3 | task 0 | new prompt, n_ctx_slot = 4096, n_keep = 0, task.n_tokens = 26
llama-swap  | srv    send_error: task id = 0, error: the current context does not logits computation. skipping
llama-swap  | slot      release: id  3 | task 0 | stop processing: n_tokens = 0, truncated = 0
llama-swap  | srv  update_slots: no tokens to decode
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  cancel_tasks: cancel task, id_task = 0
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  log_server_r: request: POST /v1/chat/completions 127.0.0.1 200
llama-swap  | [WARN] error processing streaming response: no usage or timings data found, path=/v1/chat/completions
llama-swap  | [INFO] Request 172.28.0.1 "POST /v1/chat/completions HTTP/1.1" 200 136 "Python/3.11 aiohttp/3.12.15" 5.346689607s
llama-swap  | [INFO] Request 172.28.0.1 "GET /v1/models HTTP/1.1" 200 4325 "Python/3.11 aiohttp/3.12.15" 105.789µs
llama-swap  | srv  params_from_: Chat format: Hermes 2 Pro
llama-swap  | slot get_availabl: id  2 | task -1 | selected slot by LRU, t_last = -1
llama-swap  | slot launch_slot_: id  2 | task -1 | sampler chain: logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist 
llama-swap  | slot launch_slot_: id  2 | task 3 | processing task
llama-swap  | slot update_slots: id  2 | task 3 | new prompt, n_ctx_slot = 4096, n_keep = 0, task.n_tokens = 118
llama-swap  | srv    send_error: task id = 3, error: the current context does not logits computation. skipping
llama-swap  | slot      release: id  2 | task 3 | stop processing: n_tokens = 0, truncated = 0
llama-swap  | srv  update_slots: no tokens to decode
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  cancel_tasks: cancel task, id_task = 3
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  log_server_r: request: POST /v1/chat/completions 127.0.0.1 500
llama-swap  | [WARN] metrics skipped, HTTP status=500, path=/v1/chat/completions
```
llama-swap  | [INFO] Request 172.28.0.1 "POST /v1/chat/completions HTTP/1.1" 500 114 "Python/3.11 aiohttp/3.12.15" 3.566738ms
llama-swap  | srv  params_from_: Chat format: Hermes 2 Pro
llama-swap  | slot get_availabl: id  1 | task -1 | selected slot by LRU, t_last = -1
llama-swap  | slot launch_slot_: id  1 | task -1 | sampler chain: logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist 
llama-swap  | slot launch_slot_: id  1 | task 6 | processing task
llama-swap  | slot update_slots: id  1 | task 6 | new prompt, n_ctx_slot = 4096, n_keep = 0, task.n_tokens = 201
llama-swap  | srv    send_error: task id = 6, error: the current context does not logits computation. skipping
llama-swap  | slot      release: id  1 | task 6 | stop processing: n_tokens = 0, truncated = 0
llama-swap  | srv  update_slots: no tokens to decode
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  cancel_tasks: cancel task, id_task = 6
llama-swap  | srv  update_slots: all slots are idle
llama-swap  | srv  log_server_r: request: POST /v1/chat/completions 127.0.0.1 500
llama-swap  | [WARN] metrics skipped, HTTP status=500, path=/v1/chat/completions
llama-swap  | [INFO] Request 172.28.0.1 "POST /v1/chat/completions HTTP/1.1" 500 114 "Python/3.11 aiohttp/3.12.15" 2.97489ms
llama-swap  | [INFO] Request 127.0.0.1 "GET / HTTP/1.1" 302 26 "curl/7.81.0" 36.489µs
```

**Comment by CISC** [comment]
It's a diffusion model, it's only supported by `llama-diffusion-cli` at the moment.
